thermorawfilereader/src/wrap.rs

```diff
@@ -228,18 +222,12 @@ pub struct SpectrumData<'a> {
 impl<'a> SpectrumData<'a> {
     /// The m/z array of the spectrum
     pub fn mz(&self) -> Cow<'a, [f64]> {
-        #[cfg(target_endian = "big")]
-        return Cow::Owned(self.mz.iter().copied().collect());
-        #[cfg(target_endian = "little")]
-        Cow::Borrowed(bytemuck::cast_slice(self.mz.bytes()))
+        Cow::Owned(self.mz.iter().collect())
```

Comment:
As above, but while this code is universally correct, the zero-copy alternative was important since this is one of the hottest code paths outside of the actual allocation of the memory buffer on the .NET side.

Reply:
Will revert. My C++ brain was having difficulties wrapping my head around the borrowing.